animation_nodes/base_types/tree_auto_execution.py

```diff
@@ -50,19 +50,19 @@ def getPropertiesState(self):
 
     def getProperties(self):
         self.hasError = False
-        objects = self.getObjects()
-        if len(objects) == 0 or self.dataPaths.strip() is "": return []
+        IDBlocks = self.getIDBlocks()
+        if len(IDBlocks) == 0 or self.dataPaths.strip() is "": return []
```

Comment:
hmm difficult, looks like a class name. Maybe `idBlocks` is better, or do you have another idea?

Reply:
Not sure, `idBlocks` seems ok, should I use it?

Reply:
yes